tests/test_dashboard.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+
+import utils
+from pages.dashboard import DashboardPage
+
+class DashboardPageTests(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.driver = utils.launch_driver()
+
+    def setUp(self):
+        self.dashboard_page = DashboardPage(self.driver)
+        self.dashboard_page.open()
+        self.dashboard_page.navbar.login()
+
+    def test_create_project(self):
+        project_title = 'Totally Unique Project'
+        self.dashboard_page.create_project_button.click()
+        create_project_modal = self.dashboard_page.CreateProjectModal(self.driver)
+        create_project_modal.title_input.send_keys(project_title)
+        create_project_modal.create_project_button.click()
+        project_created_modal = self.dashboard_page.ProjectCreatedModal(self.driver)
+        project_created_modal.go_to_project_button.click()
```

Comment:
Just want to make sure. Those tests aren't finished yet, right?

Reply:
Nope. I had fuller tests in the works. I just popped this part of a test into this PR to show how the class works - and check that it works for myself.